### PR TITLE
Add Simplified Chinese translation

### PR DESCRIPTION
**Pull request by nightgpt**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    <string name="system_prompt_description">系统提示词是一段命令、指导和上下文信息，在模型与用户交流前提供给它</string>
+    <string name="temperature_icon">温度的图标</string>
+    <string name="top_p_icon">核心采样的图标</string>
+    <string name="system_prompt_icon">系统提示词的图标</string>
+    <string name="temperature_setting">温度</string>
+    <string name="temperature_setting_description">温度是一个注入到回答中的随机量，温度越低，答案越集中和确定;温度越高，答案越为灵活</string>
+    <string name="top_p_setting">核心采样</string>
+    <string name="top_p_setting_description">在核心采样中，我们按递减概率顺序计算每个后续令牌的所有选项的累积分布，并在达到该值指定的特定概率后将其截断。\n
+ \n
+ 大多数模型通常建议改变这个或温度，但不是两者都改变。</string>
```
Comment (Taewan-P) [inline app/src/main/res/values-zh-rCN/strings.xml]: There is an extra newline inside this string. Is this intended? 

```suggestion
    <string name="top_p_setting_description">在核心采样中，我们按递减概率顺序计算每个后续令牌的所有选项的累积分布，并在达到该值指定的特定概率后将其截断。\n大多数模型通常建议改变这个或温度，但不是两者都改变。</string>
```
Also, please merge this string resource to one line like the above suggestion. Multiple lines inside strings.xml will not be actually shown as multiple lines in the app.

**Review by Taewan-P — CHANGES_REQUESTED** [review]

**Review by Taewan-P — APPROVED** [review]
LGTM

Thanks for the contribution!

**Comment by nightgpt** [comment]
I just installed the auto-build apk, why is it not displaying Simplified Chinese properly on my device, and still in English?
Screenshot_20240724190716
Screenshot_20240724190720

**Comment by Taewan-P** [comment]
Check for per-app language setting in your phone setting, or in your app information page. You should be able to change it there.
If your phone uses Android 12 or below, it is not currently supported.

**Comment by nightgpt** [comment]
My phone language is now Simplified Chinese, but the problem is not resolved
The "所有语言" in the screenshot means all languages, but there is no option for Simplified Chinese.
Screenshot_20240724192134

**Comment by nightgpt** [comment]
And I'm on Android 13.

**Comment by Taewan-P** [comment]
image

From my emulator, after selecting Chinese, choosing the second one on the picture above changes the app language. (Don't know how to read it) Isn't this simplified Chinese? 

image

**Comment by nightgpt** [comment]
Maybe we're not installing the same apk?
Sorry, it should be my mistake.
My apk was downloaded from this page
https://github.com/Taewan-P/gpt_mobile/actions/runs/10074482557

**Comment by nightgpt** [comment]
Or maybe it's because I'm using the Chinese version of MIUI.
The Chinese version of MIUI often has some problems

**Comment by nightgpt** [comment]
I just asked a friend who uses class native android to test the apk for me, so it should be a problem with getting the channel, sorry about that
5a57bfefed73b0e9

**Comment by nightgpt** [comment]
> image
> From my emulator, after selecting Chinese, choosing the second one on the picture above changes the app language. (Don't know how to read it) Isn't this simplified Chinese?
> 
> image

Yes.

**Comment by Taewan-P** [comment]
Is it solved?

**Comment by nightgpt** [comment]
> Is it solved?

No

**Comment by nightgpt** [comment]
Can you send me a copy of the apk file you installed?
I think it's the access channel, you probably built it yourself with the source code, while I downloaded the auto-built apk

**Comment by Taewan-P** [comment]
Looking at the workflow logs, it built the wrong source code. It should be cloning your forked version, but pulled the origin code. That's why it is not normally applied.

I can't access to my computer right now, so you should try fetching upstream of the fork or pull my repo main branch. Then, try building it on your computer & install.

**Comment by Taewan-P** [comment]
If you can't build it on your own, I might have time to upload it tonight

**Comment by nightgpt** [comment]
> Looking at the workflow logs, it built the wrong source code. It should be cloning your forked version, but pulled the origin code. That's why it is not normally applied.
> 
> I can't access to my computer right now, so you should try fetching upstream of the fork or pull my repo main branch. Then, try building it on your computer & install.

OK

**Comment by nightgpt** [comment]
> Looking at the workflow logs, it built the wrong source code. It should be cloning your forked version, but pulled the origin code. That's why it is not normally applied.
> 
> I can't access to my computer right now, so you should try fetching upstream of the fork or pull my repo main branch. Then, try building it on your computer & install.

Okay

**Comment by nightgpt** [comment]
> If you can't build it on your own, I might have time to upload it tonight

Thank you

**Comment by Taewan-P** [comment]
[app-debug.apk.zip](https://github.com/user-attachments/files/16377582/app-debug.apk.zip)

**Comment by nightgpt** [comment]
> Try this: [app-debug.apk.zip](https://github.com/user-attachments/files/16377582/app-debug.apk.zip)
> 

It’s solved.Thanks!

**Pull request merged**